services/s3/src/main/java/software/amazon/awssdk/services/s3/internal/crt/CopyObjectHelper.java

```diff
@@ -189,6 +189,9 @@ private CompletableFuture<CompleteMultipartUploadResponse> completeMultipartUplo
                                           .multipartUpload(CompletedMultipartUpload.builder()
                                                                                    .parts(parts)
                                                                                    .build())
+                                          .sseCustomerAlgorithm(copyObjectRequest.sseCustomerAlgorithm())
+                                          .sseCustomerKey(copyObjectRequest.sseCustomerKey())
+                                          .sseCustomerKeyMD5(copyObjectRequest.sseCustomerKeyMD5())
```

Comment:
Doesn't seem scalable. Is there a way to copy fields by name or something using SdkFields?

Reply:
Yeah, this was already done in https://github.com/aws/aws-sdk-java-v2/pull/4177. We can switch to that once it gets merged to mainline